package.json

```diff
@@ -20,6 +25,7 @@
     "uninstall-local": "pip uninstall dash-core-components -y",
     "build:js": "webpack --mode production",
     "build:js-dev": "webpack --mode development",
+    "build:r": "node ./extract-meta src/components > dash_core_components/metadata.json && cp package.json dash_core_components && npm run generate-python-classes && npm run generate-r-classes",
```

Comment:
Does `generate-r-classes` actually require `generate-python-classes` to be run first? regardless, the `extract-meta` and `cp package.json` parts are going to be shared so I wonder if we just want to have one command `build:backend` or something that does both languages (and any others we add later)? And use that in `build:all` (which is currently an exact copy of `prepublish` but perhaps shouldn't be cc https://github.com/plotly/dash-html-components/pull/83)

Reply:
Sorry for not replying to these comments late last month; when I last spoke with @nicolaskruchten and @T4rk1n about component generation in Python vs. R, I believe we decided that support for the latter remains experimental. 

Currently this requires setting the `--r-prefix` flag when calling `dash-generate-components`, but we could eventually remove the flag and generate both at once. 

It does (for now) require that the Python classes are generated first, since JS dependency information is sourced from `__init__.py` and the R package version number from `version.py`.

Reply:
remind me again why the R transpiler gets the version info from a generated Python file? I know we've talked about this before but I don't recall.